src/index.html

```diff
@@ -6,8 +6,38 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main
+    class="card"
+    data-qa="card">
+      <div class="card__title-container">
+        <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
```

Comment:
[Use semantic tags where possible](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-content)

These tags `h2-h6` are great for this case